src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryDataLoader.kt

```diff
@@ -81,7 +83,7 @@ object ChocolateFactoryDataLoader {
     )
     private val timeTowerStatusPattern by ChocolateFactoryAPI.patternGroup.pattern(
         "timetower.status",
-        "§7Status: §.§l(?<status>INACTIVE|ACTIVE)(?: §f)?(?<acitveTime>\\w*)",
+        "§7Status: §.§l(?<status>INACTIVE|ACTIVE)(?: §f)?(?<activeTime>\\w*)",
```

Comment:
we cant change this regex without also changing the key of this pattern. otherwise older mod versions would break.

Reply:
Would changing the key or reverting to the typo-ed capture name be ideal in this case, then?

Reply:
Reverted to pre-PR state in latest commit.